src/pseudopeople/entity_types.py

```diff
@@ -36,7 +36,7 @@ def default_noise_level_getter(
 @dataclass
 class NoiseType(ABC):
     name: str
-    noise_function: Callable = _noise_function_not_implemented
+    noise_function: Callable[..., None] = _noise_function_not_implemented
```

Comment:
Can't we be more precise about the arguments a noise function expects?

Reply:
We can be precise in the subclasses RowNoiseType and ColumnNoiseType but the default argument to noise_function has the signature def _noise_function_not_implemented(*_args: Any, **_kwargs: Any).